packages/design-system/src/molecules/sidebar/mobile.tsx

```diff
@@ -41,11 +41,13 @@ const styles = StyleSheet.create({
 
 export const MobileSidebar: React.FC<SidebarProps> = ({
   logo,
-  withConnectWallet = false,
   header,
   footerOptionsOnMobile,
   sections,
   selectedOptionId,
+  mobileSuffix,
+  mobileSidebarSuffix,
```

Comment:
The same thing here about the `mobile` keyword being used in the prop names

```suggestion
  suffix,
  sidebarSuffix,
```

Reply:
the thing here is that both components desktop and mobile share the same props `SidebarProps` maybe we can do something similar we did with the logo 

```
export interface LogoSidebar {
  desktop: React.ReactNode
  mobileHeader: React.ReactNode
  mobileSidebar: React.ReactNode
}